tests/unit/lms/services/aes_service_test.py

```diff
@@ -0,0 +1,25 @@
+from unittest.mock import sentinel
+
+import pytest
+
+from lms.services.aes import AESService, factory
+
+
+class TestAESService:
+    @pytest.mark.parametrize("text, expected", [(b"TEXT", b"TEXT"), ("TEXT", b"TEXT")])
+    def test_it(self, svc, text, expected):
```

Comment:
Hmm... :thinking: I think I'd call this a "round-trip" test, and I think it's valuable to have, but what's missing here is that you could replace the encrypt and decrypt functions with `lamdba value: value` and it would still pass.

It would be good (and quite easy) to capture some canned responses for the encrypt and decrypt functions. You could also assert the `build_iv` gives you something of the expected length at least.